bezier/src/lib.rs

```diff
@@ -247,6 +253,30 @@ impl QuadraticBezierSegment {
         });
         return len;
     }
+
+    pub fn bounding_rect(&self) -> Rect {
+        let min_x = self.from.x.min(self.ctrl.x).min(self.to.x);
+        let max_x = self.from.x.max(self.ctrl.x).max(self.to.x);
+        let min_y = self.from.y.min(self.ctrl.y).min(self.to.y);
+        let max_y = self.from.y.max(self.ctrl.y).max(self.to.y);
+
+        return euclid::rect::rect(min_x, min_y, max_x - min_x, max_y - min_y);
```

Comment:
nit: just `use euclid::rect::rect` at the top of this file and then call `rect(...`.